Sorting/SortingUtils.h

```diff
@@ -9,15 +9,24 @@
 
 #ifndef VECTOR_H
 #include <vector>
-#endif
+#include <cstdlib>
 
+#endif
+#define EXIT_CODE_ERROR -1
 using namespace std;
 
 // Input utils:
 
 void getInputSize(size_t& size) {
     cout << "Enter the input size : ";
     cin >> size;
+    if(size <0) {
```

Comment:
Since `size` is of type `size_t`, which is an unsigned integer, it will never _actually_ be negative, and this condition will always be false!
Eg. if the value entered for `size` is `-5`, in binary it will be (assuming an `int` is 4 bytes long) `10000000000000000000000000000101` which for a `size_t` in decimal is `2147483653`.

It would therefore be right to change this line to:
```C++
    if ((int) size < 0) {
```

What are your thoughts on this?